src/pages/library/dataBrowser-utils.js

```diff
@@ -21,22 +21,13 @@ export const uiMessaging = {
 }
 
 export const datasetReleasePolicies = {
-  'TerraCore:NoRestriction': { label: 'NRES', desc: 'No restrictions' },
-  'TerraCore:GeneralResearchUse': { label: 'GRU', desc: 'General research use' },
-  'TerraCore:NPOA': { label: 'NPOA', desc: 'No population origins or ancestry research' },
-  'TerraCore:NMDS': { label: 'NMDS', desc: 'No general methods research' },
-  'TerraCore:GSO': { label: 'GSO', desc: 'Genetic studies only' },
-  'TerraCore:CC': { label: 'CC', desc: 'Clinical care use' },
-  'TerraCore:PUB': { label: 'PUB', desc: 'Publication required' },
-  'TerraCore:COL': { label: 'COL', desc: 'Collaboration required' },
-  'TerraCore:IRB': { label: 'IRB', desc: 'Ethics approval required' },
-  'TerraCore:GS': { label: 'GS', desc: 'Geographical restriction' },
-  'TerraCore:MOR': { label: 'MOR', desc: 'Publication moratorium' },
-  'TerraCore:RT': { label: 'RT', desc: 'Return to database/resource' },
-  'TerraCore:NCU': { label: 'NCU', desc: 'Non commercial use only' },
-  'TerraCore:NPC': { label: 'NPC', desc: 'Not-for-profit use only' },
-  'TerraCore:NPC2': { label: 'NPC2', desc: 'Not-for-profit, non-commercial use only' },
-  releasepolicy_other: { policy: 'SnapshotReleasePolicy_Other', label: 'Other', desc: 'Misc release policies' }
+  'DUO:0000007': { label: 'DS', desc: 'disease specific research' },
+  'DUO:0000042': { label: 'GRU', desc: 'General research use' },
+  'DUO:0000006': { label: 'HMB', desc: 'Health or medical or biomedical research' },
+  'DUO:0000011': { label: 'POA', desc: 'Population origins or ancestry research only' },
+  'DUO:0000004': { label: 'NRES', desc: 'No restriction' },
+  // This case can handle null policies, or one that has been added to the schema enum that we don't yet handle
+  unknownReleasePolicy: { policy: 'Unknown', label: 'Unknown', desc: 'No provided dataset release policy or not yet handled in the web interface' }
```

Comment:
I think it would be better to show an unknown policy differently than a missing policy. An unknown policy can still be shown in the UI, although we don't know its description and we'd have to use its value as the label.

Reply:
Sure! I can change that